packages/maru-ui/src/Text/Text.tsx

```diff
@@ -0,0 +1,23 @@
+import { HTMLAttributes, ReactNode } from 'react';
+import { font, color } from '@maru/theme';
+import styled from 'styled-components';
+
+interface PropsType extends HTMLAttributes<HTMLSpanElement> {
+    children: ReactNode;
+    color?: string;
+    fontType: keyof typeof font;
+}
+
+const Text = ({ children, color, fontType }: PropsType) => {
+    return (
+        <StyledText style={{ color }} fontType={fontType}>
+            {children}
+        </StyledText>
+    );
+};
+
+export default Text;
+
+const StyledText = styled.p<{ fontType: keyof typeof font }>`
```

Comment:
`keyof typeof font` 를 따로 타입으로 빼는게 어떨까요? 두 개 밖에 없더라도 좀 더 깔끔해 보일 것 같아요

```suggestion
type FontType: keyof typeof font;

interface PropsType extends HTMLAttributes<HTMLSpanElement> {
    children: ReactNode;
    color?: string;
    fontType: FontType;
}

const Text = ({ children, color, fontType }: PropsType) => {
    return (
        <StyledText style={{ color }} fontType={fontType}>
            {children}
        </StyledText>
    );
};

export default Text;

const StyledText = styled.p<{ fontType: FontType }>`
```

Reply:
음 사실 그렇게 한 이유가 font가 언제든지 추가되어도 type으로 따로 지정해주지않아도 type으로 들어가서 해주었던건데 명재님 의견이 맞는것같습니다 따로 뺴는게 좋을것같아요 감사합니다~